scripts/release.ts

```diff
@@ -84,13 +84,14 @@ async function prepareRelease(opts: BuildOptions, args: ReadonlyArray<string>, r
     },
     {
       type: 'input',
-      name: 'version',
-      message: 'Version',
+      // this name is intentionally different from 'version' above. if they collide, this input prompt will not run.
```

Comment:
```suggestion
      // this name is intentionally different from 'version' above to make the `when` check below work properly
      // (this prompt should only run if `version` was not already input)
```

is this an accurate re-phrasing of this comment here? it looks like the `when` check doesn't check that the values are the same, but rather that `version` was not set in the above prompt